lib/runners/processor/pylint.rb

```diff
@@ -0,0 +1,91 @@
+module Runners
+  class Processor::Pylint < Processor
+    Schema = StrongJSON.new do
+      let :runner_config, Schema::BaseConfig.base.update_fields { |fields|
+        fields.merge!(
+          rcfile: string?,
+          ignore: enum?(string, array(string)),
+          errorsOnly: boolean?,
```

Comment:
Please fix it according to the convention between other tools:

```suggestion
          errors-only: boolean?,
```

Reply:
Sorry, quotes are needed:

```suggestion
          "errors-only": boolean?,
```

Usage:

```ruby
config_linter[:"errors-only"]
```